.spelling

```diff
@@ -3,3 +3,4 @@ kickstart
 5pm
 75mm
 SKL_ADDR
+9am
```

Comment:
nit: shall we aim to keep this file (sort-of) alphabetically sorted?

Reply:
I recall a similar conversation took place for SourceBots last year. I suggest we look at the conclusion for that.

Reply:
We probably should. Seeing as the file isn't in order outside this PR, let's ship this one, and reorder it then, as there probably won't be any open PRs then

Reply:
It's _almost_ in alphabetical order. I was imagining that we could insert `9am` one line higher and it would still be _mostly_ in order.